commercial/app/views/static/subscriberNumberPage.scala.html

```diff
@@ -0,0 +1,35 @@
+@(metaData: model.MetaData)(implicit request: RequestHeader)
+
+    @main(model.SimplePage(metaData)) { } {
+        <div class="l-side-margins">
+            <div class="live-events-survey-wrapper" data-link-name="live events survey page">
+                <div class="live-events-survey-wrapper-inner">
+                    <h1>Enter subscriber number</h1>
+                    <p class="standfirst">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus metus massa, convallis eget pellentesque sollicitudin, rhoncus vitae sem. Cras non lorem sit amet orci pellentesque tincidunt ut eget tellus. Donec et orci dictum, aliquam sapien eget, porta felis. Integer ut feugiat odio. Sed vulputate nisi tortor, nec semper enim posuere sed. Nullam eu maximus arcu. Donec rutrum ex a dignissim dapibus.</p>
+                    <div class="subscriber-number">
+                        <form class="subscriber-number-form js-subscriber-number-form">
+                            <input type="text" name="number" id="number" class="input-number"/>
+                            <button type="submit" name="submit" class="button--primary">Submit</button>
+                        </form>
+                        <p class="subscriber-number-correct standfirst js-subscriber-number-correct u-h">
+                            Your subscriber number is correct! Continue reading your previously requested page: <a href="https://membership.theguardian.com?INTCMP=DISMISS_COMMERCIAL_SURVEY_MEMBERSHIP" class="" data-link-name="article title">Article title</a>
+                        </p>
+                        <p class="subscriber-number-incorrect standfirst js-subscriber-number-incorrect u-h">
+                            Your subscriber number is incorrect! In order to continue reading content on The Guardian <a href="">join Membership</a> or <a href="">whitelist our page</a>.
```

Comment:
Yes, I just wanted to deliver the logic now. The whole text and links may and probably will change.